documentation/modules/metrics/proc_metrics-deploying-prometheus.adoc

```diff
@@ -34,7 +34,10 @@ sed -i '' 's/namespace: .*/namespace: _my-namespace_/' prometheus.yaml
 . Edit the `PodMonitor` resource in `strimzi-pod-monitor.yaml` to define Prometheus jobs that will scrape the metrics data from pods.
 `PodMonitor` is used to scrape data directly from pods and is used for Apache Kafka, ZooKeeper, Operators, and Kafka Bridge. Update the `namespaceSelector.matchNames` property with the namespace where the pods to scrape the metrics from are running.
 
-. To use another role:
+. The provided Grafana dashboards show metrics about CPU, memory and volume disks usage.
+These metrics come from cadvisor and kubelet directly on the nodes.
+In order to have Prometheus scraping for these metrics, an additional configuration is needed.
+This is because the Prometheus operator doesn't have a corresponding of `PodMonitor` for scraping metrics coming from nodes directly.
```

Comment:
```suggestion
This is because the Prometheus operator does not have a corresponding of `PodMonitor` for scraping metrics coming from nodes directly.
```

Reply:
Thanks Standa!